openmls/src/ciphersuite/signature.rs

```diff
@@ -188,7 +197,8 @@ impl SignaturePrivateKey {
     }
 
     /// Returns the raw private key bytes as slice.
-    pub(crate) fn as_slice(&self) -> &[u8] {
+    #[cfg(feature = "crypto-subtle")]
+    pub fn as_slice(&self) -> &[u8] {
```

Comment:
Is it really necessary to expose this? Consumers who want to bring their own private keys can do so already. Maybe at this point we should add a chapter on the `crypto-subtle` API to the book. Just to keep an overview over what it enables.